lib/tasks/release_note.rake

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
-DATE_REGEX = /\A\d{4}-\d{2}-\d{2}/
+DATE_REGEX ||= /\A\d{4}-\d{2}-\d{2}/
```

Comment:
In the past @tgxworld has been against fixing errors this way and preferred us to not include the same file twice. Did you investigate a solution that prevented it from being double loaded first?

Reply:
Rake tasks are loaded every time we call `Discourse::Application.load_tasks` which we do in 7 `before` blocks in specs. Two of those are unnecessary but the other 5 (combined with `Rake::Task.clear`) are required to test those tasks.

Reply:
I see. This is a problem with the way rake works and there isn't really anything we can do about it. Thanks for the explanation! I will approve now.